src/game/engine/game.py

```diff
@@ -3,34 +3,45 @@
 from game.style import color as cval
 from game.style import text
 
+from .player import Player
+
+
 def draw_text(text, font, color, surface, x, y):
     textobj = font.render(text, 1, color)
     textrect = textobj.get_rect()
     textrect.topleft = (x, y)
     surface.blit(textobj, textrect)
 
+
 def game(screen, main_clock):
     running = True
-    
+
     print("FROM GAME: running =", running)
+    player = Player((0, 400, 50, 50), "path_to_player_image")
+
     while running:
         running = True
-    
+
         screen.fill(cval.black)
-       
-        draw_text('game', text.default_font, cval.white, screen, 20, 20)
+
+        draw_text("game", text.default_font, cval.white, screen, 20, 20)
         for event in pygame.event.get():
             if event.type == pygame.QUIT:
                 pygame.quit()
                 sys.exit()
             if event.type == pygame.KEYDOWN:
                 if event.key == pygame.K_ESCAPE:
                     running = False
-        
+                if event.key == pygame.K_RIGHT:
+                    player.right()
```

Comment:
If my understanding of the game is right, we'll never really need this as a core mechanic, right? If this just for debugging?

Reply:
cool, yeah just proof of concept